src/PVRIptvData.cpp

```diff
@@ -1482,3 +1487,28 @@ int PVRIptvData::GetChannelId(const char * strChannelName, const char * strStrea
 
   return abs(iId);
 }
+
+std::string& PVRIptvData::AddHeaderToStreamUrl(std::string& strLine, const std::string& header)
+{
+  size_t found = strLine.find('|');
+
+  if (found != std::string::npos)
+  {
+    size_t present = strLine.find(header + '=', found + 1);
```

Comment:
Why not put the find call in the if statement as you don't use the `present` variable other than in the if.

Reply:
I can save one variable and a line putting inside the if, for sure. Did with the purpose to make the lines shorter.